<launch>
    <!-- Elevation mapping node -->
    <node pkg="elevation_mapping_cupy" type="elevation_mapping_node" name="elevation_mapping" output="screen">
        <rosparam command="load" file="$(find elevation_mapping_cupy)/config/core/core_param.yaml"/>
        <rosparam command="load" file="$(find elevation_mapping_cupy)/config/core/example_setup.yaml"/>
    </node>
</launch>